src/core_plugins/kibana/public/home/components/feature_directory.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Synopsis } from './synopsis';
+import {
+  KuiTabs,
+  KuiTab,
+  KuiFlexItem,
+  KuiFlexGrid,
+} from 'ui_framework/components';
+
+const ALL = 'all';
+
+export class FeatureDirectory extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    const tabs = [{
+      id: ALL,
+      name: 'All',
+    }];
+    if (props.directories.some(directory => directory.category === props.directoryCategories.DATA)) {
+      tabs.push({
+        id: props.directoryCategories.DATA,
+        name: 'Explore & Visualize'
+      });
+    }
+    if (props.directories.some(directory => directory.category === props.directoryCategories.ADMIN)) {
+      tabs.push({
+        id: props.directoryCategories.ADMIN,
+        name: 'Administrative',
+      });
+    }
+    if (props.directories.some(directory => directory.category === props.directoryCategories.OTHER)) {
+      tabs.push({
+        id: props.directoryCategories.OTHER,
+        name: 'Other',
+      });
+    }
+
+    this.tabs = tabs;
+    this.state = {
+      selectedTabId: ALL
+    };
+  }
+
+  onSelectedTabChanged = id => {
+    this.setState({
+      selectedTabId: id,
+    });
+  };
+
+  renderTabs = () => {
+    return this.tabs.map((tab, index) => (
+      <KuiTab
+        onClick={() => this.onSelectedTabChanged(tab.id)}
+        isSelected={tab.id === this.state.selectedTabId}
+        key={index}
+      >
+        {tab.name}
+      </KuiTab>
+    ));
+  }
+
+  renderDirectories = () => {
+    return this.props.directories.inTitleOrder
+    .filter((directory) => {
```

Comment:
nit, think the .filter and .map functions should be indented by two because it's a line continuation.